README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
-- If you have stock, what ultimately determines its value is **percentage ownership** of the entire company, not the absolute number of shares. To determine the percentage of the company a certain number of shares represents, divide it by the number of outstanding shares.
-- ☝️🔹 However, there are subtleties to be aware of regarding what this outstanding total refers to:
-  - Private companies always have what is referred to as **“authorized but unissued”** shares. For example, a corporation might have 100 million authorized shares, but will only have actually issued 10 million shares. In this example, the corporation would have 90 million authorized but unissued shares. When you are trying to determine what percentage a number of shares represents, you do *not* make reference to the authorized but unissued shares.
-  - You actually want to know the total issued, but even this number can be confusing, as it can be computed [more than one way](http://www.mattbartus.com/option-grants-fully-diluted-or-issued-and-outstanding/). Typically, people refer to the total number of shares “issued and outstanding” or “fully diluted.”
-  - **“Issued and outstanding”** refers to the number of shares actually issued by the company to shareholders. Note this will not include shares that others may have an option to purchase.
-  - **“Fully diluted”** refers to all of the shares that have been issued, all of the shares that have been set aside in a stock incentive plan, and all of the shares that could be issued if all convertible securities (such as outstanding warrants) were exercised. A key difference is that this total will include all the shares in the employee option pool that are reserved but not yet issued to employees. (The option pool is discussed more below.)
-  - Generally, it’s best to know the fully diluted number to know the likely percentage a number of shares is worth in the future. The terminology mentioned here isn’t universally applied, either, so it’s worth discussing it to be sure there is no miscommunication.
+- [**Public companies**](https://en.wikipedia.org/wiki/Public_company) are corporations where any member of the public can own stock. People can buy and sell the stock for cash on public [exchanges](https://www.investopedia.com/terms/e/exchange.asp). The value of a company’s shares is the value you see in the stock market reports, so shareholders know how much their stock is worth.
+- Most smaller companies, including all startups, are [**private companies**](https://en.wikipedia.org/wiki/Privately_held_company), where shareholders are restricted, meaning you can’t buy and sell stock easily or to anyone at any time.
```

Comment:
This description of a private company feels light. The crux of a private company is that the owners of the company can control who is able to buy and sell their stock, vs. a public company where anyone is free to buy and sell.

Reply:
That's a nice elaboration. Of course that "control" is only real insofar as a company is in a position to fully control their funding.

Reply:
✔️